appservice/package.json

```diff
@@ -40,9 +40,9 @@
         "@azure/core-client": "^1.7.2",
         "@azure/core-rest-pipeline": "^1.10.3",
         "@azure/storage-blob": "^12.3.0",
-        "@microsoft/vscode-azext-azureutils": "^2.0.6",
+        "@microsoft/vscode-azext-azureutils": "2.1.0",
```

Comment:
Is there a reason to not continue using caret notation? i.e.:
```suggestion
        "@microsoft/vscode-azext-azureutils": "^2.1.0",
```

Reply:
No. I forgot to put it back when I stopped using the beta versions. 🙈